mobile/src/libs/components/loader/loader.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ActivityIndicator } from 'react-native';
+
+import { AppColor } from '#libs/enums/enums';
+
+const Loader: React.FC = () => {
+  return <ActivityIndicator size="large" color={AppColor.BLUE_100} />;
```

Comment:
As @fshabanov mentioned, please wrap it into View and add styles so it will be shown at the center

Reply:
Done, thank you.
<img width="297" alt="Screenshot 2023-08-20 at 14 33 58" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/4521937/d9c987ee-04d1-4db9-ab25-26072634aa73">